### ISSUE TITLE
Does OpenRoad support UPF function now?

### ISSUE THREAD

**Issue by chros098** [The-OpenROAD-Project/OpenROAD-flow-scripts#2611]
### Description

I would like to ask if the openroad flow supports upf now? I saw that there are related components under openroad, but I did not find this read process in the makefile of flow.

**Comment by maliberty** [comment]
UPF is a massive standard.  We support a limited/useful subset.  We have no designs that have UPF to put into ORFS but would be glad to have a public example.

**Comment by chros098** [comment]
We use openroad, upf that component, use our own process, and then refer to the upf example, run directly crash, do not know where the problem is, if there is a problem with the upf, it will report the problem, no problem, openroad gui directly crash

**Comment by chros098** [comment]
I'm glad you replied to me. I'm also thinking about how to implement UPF.

**Comment by maliberty** [comment]
That's a bit hard to parse.  UPF hasn't had a lot of use yet so I wouldn't be surprised if there are issues.  We are glad to address them with the usual caveat about priority.  PRs with fixes/improvements are welcome.

**Comment by chros098** [comment]
OK, thanks. I'll close the question.
